src/lib/server/websearch/runWebSearch.ts

```diff
@@ -13,12 +13,21 @@ import { defaultEmbeddingModel, embeddingModels } from "$lib/server/embeddingMod
 const MAX_N_PAGES_SCRAPE = 10 as const;
 const MAX_N_PAGES_EMBED = 5 as const;
 
-const DOMAIN_BLOCKLIST = ["youtube.com", "twitter.com"];
+import { WEBSEARCH_ALLOWLIST, WEBSEARCH_BLOCKLIST } from "$env/static/private";
```

Comment:
small nit: the imports should be on top of the file. Otherwise, having 2 import blocks looks weird

Reply:
cleaned it up!